content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,269 @@
+the coverage of our nightly tests that we are able to maintain. Maintaining
+a large range of Kubernetes versions is tricky, that is why some versions
+are in the column "Untested but may work". The configuration for the
+nightly can be seen in [`previous-periodics.yaml`][] and
+[`next-periodics.yaml`][], and the results are visible at
+[`testgrid-previous`][] and [`testgrid-next`][]).
+
+[next-periodics.yaml]: https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/release-next/cert-manager-release-next-periodics.yaml
+[previous-periodics.yaml]: https://github.com/jetstack/testing/blob/master/config/jobs/cert-manager/release-previous/cert-manager-release-previous-periodics.yaml
+[testgrid-previous]: https://testgrid.k8s.io/jetstack-cert-manager-previous
+[testgrid-next]: https://testgrid.k8s.io/jetstack-cert-manager-next
+
+#### Security issues {#security-issues}
+
+**Security issues** are fixed as soon as possible. They get back-ported to
+the last two releases, and a new patch release is immediately created for
+them:
+
+```text
+  v1.1.0                                           v1.1.1
+----+-------------------------------------------------+-----> release-1.1
+     \                                       backport ^
+      \                                      commit  /
+       \                                            /
+        \                                          /
+         \                                        /
+          \         v1.2.0               v1.2.1  /
+           ------------+---------------------+--/-----------> release-1.2
+                        \           backport ^ /
+                         \          commit  / /
+                          \                / /
+                           \              / /
+                            \            / /
+                             \          / /
+                              ------------------------------> master
+                                     ^   ^
+                            security |   | two new patch
+                               patch |   | releases
+                              merged       created immediately
+```
+
+<!-- Diagram source: https://textik.com/#7c4096204b3c0ad3 -->
+
+#### Critical bugs {#critical-bugs}
+
+**Critical bugs** include both regression bugs as well as upgrade bugs.
+
+Regressions are functionalities that did work in the previous release but
+no longer works. [#3393][] and [#2857][] are two examples of regressions.
+
+Upgrade bugs are bugs (often Helm-related) preventing users from upgrading
+to earlier final releases of cert-manager. [#3882][] and [#3644][] are
+example of such upgrade bugs.
+
+The fixes are back-ported; we then wait until the next final release for
+trigger a patch release. The patch release will be made for all supported
+release. Note that since we do only support the two last releases, patch
+releases are only made to the last release.
```

Comment:
question: is this saying that we _don't_ create a new patch release after backporting the fix? if so, that seems wrong to me?

suggestion: grammar/rewording. I also personally find this quite confusing to read and parse, but I held off from suggesting wider changes to the wording since I don't fully understand the process we're describing here yet.

```suggestion
The fixes are back-ported and we then wait until the next final release before
triggering a patch release. The patch release will be made for all supported
releases, but note that since we only support the two last releases, patch
releases are only made to the last release.
```

Reply:
> is this saying that we don't create a new patch release after backporting the fix?

Yes, that's what @munnerz suggested in https://github.com/cert-manager/website/pull/452#discussion_r610496126; maybe I should just say this:

> The fixes related to critical bugs are back-ported "in batch": the patch release is created along with the next final release (we do not create a patch release for a single critical bug fix).

Reply:
I rephrased the paragraph in ae37d5e, let me know if you feel it is still confusing.